scalafix-tests/shared/src/main/scala/test/organizeImports/Case.scala

```diff
@@ -0,0 +1,19 @@
+package test.organizeImports
+
+object Case {
```

Comment:
Could you test/document where literals starting with characters between uppercase and lowercase in the ASCII table (like `_foo`) end up?